app/assets/javascripts/modules/ukvi-ab-test-september-2016.js

```diff
@@ -0,0 +1,53 @@
+//= require govuk/multivariate-test
+
+var UkviABTest = UkviABTest || {};
+
+UkviABTest.isSection = function(pathname) {
+  return window.location.pathname == pathname;
+}
+
+UkviABTest.AddtoApplySection = function() {
+  var html = `<p>You can <a href='https://visas-immigration.service.gov.uk/product/eea-qp'>apply online as a qualified person</a> but not if you’re a student or self-sufficient person and you're either:
```

Comment:
Template Literals are an ES6 feature, so we'll have to do it like

```js
var html = '' +
'<h1>Hello, World</h1>' +
'<p>Test</p>' +
''
```

or

```js
var html = [
  '<h1>Hello, World</h1>',
  '<p>test</p>'
].join('')
```
(Hopefully we'll be able to think about transpiling ES6 in the future though since it's a much nicer syntax for things like this!)

http://caniuse.com/#feat=template